openvdb/openvdb/CMakeLists.txt

```diff
@@ -455,26 +465,7 @@ endif()
 
 # Private defines (not inherited by dependent projects)
 list(APPEND OPENVDB_CORE_PRIVATE_DEFINES -DOPENVDB_PRIVATE)
-if(USE_BLOSC)
-  list(APPEND OPENVDB_CORE_PRIVATE_DEFINES -DOPENVDB_USE_BLOSC)
-endif()
-
-if(USE_BLOSC OR USE_ZLIB)
-  list(APPEND OPENVDB_CORE_PRIVATE_DEFINES -DOPENVDB_USE_ZLIB)
-endif()
 
-# Public defines
-if(WIN32)
-  # @note OPENVDB_OPENEXR_STATICLIB is old functionality from the makefiles
-  #       used in PlatformConfig.h to configure EXR exports. Once this file
-  #       is completely removed, this define can be too
-  if(USE_IMATH_HALF)
-    if(ILMBASE_USE_STATIC_LIBS OR (${ILMBASE_Half_LIB_TYPE} STREQUAL STATIC_LIBRARY))
-      list(APPEND OPENVDB_CORE_PUBLIC_DEFINES -DOPENVDB_OPENEXR_STATICLIB)
-    endif()
-  endif()
-  list(APPEND OPENVDB_CORE_PUBLIC_DEFINES -D_WIN32 -DNOMINMAX)
```

Comment:
This changes behaviour when building VDB. Previously these were provided directly where as now they are defined in `Platform.h`. This means that the header include order for VDB is important (and is the reason why so many of the Test files have needed updating as `gtest` includes the windows/standard files which use these defines)